topics/homework_topic2_Lobashova/frozenset.py

```diff
@@ -0,0 +1,43 @@
+# Initial steps/scenarios
+steps_1 = ["open browser", "navigate to page", "click login"]
+steps_2 = ["open browser", "navigate to page", "fill form"]
+steps_3 = ["open browser", "navigate to page", "click login", "click log out"]
+
+# Convert each scenario's steps to frozensets
+scenario_1 = frozenset(steps_1)
+scenario_2 = frozenset(steps_2)
+scenario_3 = frozenset(steps_3)
+
+# Dictionary of scenarios with frozensets
+dic_scenarios = {
+    "Test Case 1": scenario_1,
+    "Test Case 2": scenario_2,
+    "Test Case 3": scenario_3
+}
+
+# Function to check if a new test scenario already exists
+#If we want to see only exact matches:
+def check_exact_matches(test_name, new_steps, existing_scenarios):
```

Comment:
instead of comment is better to use """Function description""" at the beginning of function. It can be used as print(check_exact_matches.__doc__)